code/API_definitions/kyc-fill-in.yaml

```
@@ -49,25 +49,19 @@ info:
 
     It is important to remark that in cases where personal user data is processed by the API, and users can exercise their rights through mechanisms such as opt-in and/or opt-out, the use of 3-legged access tokens becomes mandatory. This measure ensures that the API remains in strict compliance with user privacy preferences and regulatory obligations, upholding the principles of transparency and user-centric data control.
 
-
-  termsOfService: http://swagger.io/terms/
-  contact:
-    email: project-email@sample.com
   license:
     name: Apache 2.0
     url: https://www.apache.org/licenses/LICENSE-2.0.html
 
   version: wip
+  x-camara-commonalities: 0.4.0
 
 servers:
-  - url: '{apiRoot}/{basePath}'
+  - url: '{apiRoot}/kyc-fill-in/v0.1'

```

Aligned with the version field, that would be currently. The actual version should be set in the Release PR (I suppose it will be 0.2.0-rc.1 and therefore the url '{apiRoot}/kyc-fill-in/v0.2rc1'

```suggestion
  - url: '{apiRoot}/kyc-fill-in/vwip'

```